src/coreclr/inc/shash.h

```diff
@@ -298,7 +323,7 @@ class EMPTY_BASES_DECL SHash : public TRAITS
     // it is perfectly fine for the element to be a duplicate - if so it
     // is added an additional time. Returns TRUE if a new empty spot was used;
     // FALSE if an existing deleted slot.
-    static BOOL Add(element_t *table, count_t tableSize, const element_t &element);
+    BOOL Add(element_t *table, count_t tableSize, const element_t &element) const;
```

Comment:
This private overload was previously static, I changed a couple of static functions to be instance functions with const because they needed to call `RemoveElement`, which is an instance function, to support autoremove. In this case it would make sense to make it non-const in its implementation, fixed.